lib/DirectEditing/TextDirectEditor.php

```diff
@@ -159,6 +164,14 @@ public function open(IToken $token): Response {
 				'session' => \json_encode($session->getData())
 			]);
 			$this->initialStateService->provideInitialState('directEditingToken', $token->getToken());
+			$this->initialStateService->provideInitialState(
```

Comment:
Maybe we can just reuse the existing [InitialStateProvider](https://github.com/nextcloud/text/blob/130a4ec5fe5f24e0e27a63e459eeea22472770e5/lib/Service/InitialStateProvider.php#L20) class for this?

The textprocessing state would also be useful for direct editing.